docs/how-to-guides/platform-integration/vtex/implementing-order-form-sync.md

```diff
@@ -0,0 +1,193 @@
+        variant="tertiary"
+        icon={<Icon name="User" width={18} height={18} weight="bold" />}
+
+```
+#### `src/pages/account.tsx` 
+Responsible for rendering the My Account page. Add the following to allow subsequent pages to reference the the user’s session cookie of this page:
+```JavaScript
+
+function Page() {
+  useEffect(() => {
+  +  window.location.href = `${storeConfig.accountUrl}${window.location.search}`
+  }, [])
+
+  return (
+
+```
+
+#### `src/pages/login.tsx` 
+Responsible for rendering the Login page. Add the following to allow subsequent pages to reference the the user’s session cookie of this page:
+
+```JavaScript
+
+function Page() {
+  useEffect(() => {
++    window.location.href = +`${storeConfig.loginUrl}${window.location.search}`
+  }, [])
+
+  return (
+
+```
+### Step 3: Update other portal environments (Optional)
+If your store uses other environments on the portal that do not have the `/api/io` route in the url, you must also pass the order form parameter to these pages when the user navigates from the store in Fast Store to these links and add this script at those pages:
+
+```JavaScript
+<script>
+    let params = new URLSearchParams(document.location.search);
+    let orderId = params.get("orderFormId");
+  
+    function callUrlWithRetry(url, maxRetries, retryInterval) {
+      return fetch(url, {
+        method: "POST",
+      })
+        .then((response) => {
+          if (response.ok) {
+            return;
+          } else {
+            throw new Error(`Without vtex_session cookie`);
+          }
+        })
+        .catch((error) => {
+          if (maxRetries > 0) {
+            return new Promise((resolve) => {
+              setTimeout(() => {
+                resolve(callUrlWithRetry(url, maxRetries - 1, retryInterval));
+              }, retryInterval);
+            });
+          } else {
+            throw error;
+          }
+        });
+    }
+  
+    callUrlWithRetry(`/api/io/faststore/cookie`, 3, 5000);
+    if (orderId) {
+      callUrlWithRetry(
+        `/api/checkout/pub/orderForm/${orderId}?refreshOutdatedData=true`,
+        3,
+        5000
+      );
+    }
+  </script>
+
+
+```
+## Creating Session Cookie in FastStore Store for Localhost or Internal URLs
+The service app generates the session cookie on the main domain. Still, if the store is using localhost or an internal URL at the checkout, it is necessary to create the cookie manually in the faststore store. To do so, navigate to the checkout URL and copy the cookie value. For example:
+
+![session-cookie-example](https://user-images.githubusercontent.com/67270558/232832503-e1426219-caa2-487d-a220-1c26a8226bbf.png)
+
+Go back to the Fast Store store and use the following command to generate the cookie that will be used to sync the `orderForm`:
+
+```JavaScript
+document.cookie = "vtex_session={{value of session cookie}}";
+
+```
+After that, you should see the cookie created and the order form synced.
```

Comment:
⛏️ Nitpicking: the same thing related to the pattern: `order form`, `OrderForm`, `orderForm`.